### ISSUE TITLE
To what correspond the HiFi-GAN losses?

### ISSUE THREAD

**Issue by abaddon-moriarty** [TensorSpeech/TensorFlowTTS#635]
Hi, 
I'm trying to figure out, what all the losses in the HG model are, and what their ideal value should be, or what they sould go towards. 

![image](https://user-images.githubusercontent.com/56978503/127474696-e0ba5441-3de1-4ffd-914b-a319e308b8ef.png)
![image](https://user-images.githubusercontent.com/56978503/127474723-6340dc1c-87cc-4578-9b86-b13647d8f496.png)

From what I could find:

- the fm_loss which is I think the feature matching loss (?) is the one described in the paper: ![image](https://user-images.githubusercontent.com/56978503/127475119-0c54373e-5d3b-45ef-9d19-45881aa21d59.png)
calculated from the intermediate features of the discriminator and its distance to ground truth

- the spectral convergence loss is replacing the mel-spectrogram loss to reuse the melgan trainer.

- gen_loss (is the generator loss ?) and 
![image](https://user-images.githubusercontent.com/56978503/127476646-1e50243e-34a3-4eda-bfef-519588f55683.png)
but then mine started at .6 for eval and .4 for training and is above 2 for both.

and I couldn't find any indication on the others. Does anyone have an idea on where I could find them.

Thank you for your answers.

**Comment by abaddon-moriarty** [comment]
@dathudeptrai , @ZDisket Did you find any information ?
